index.html

```diff
@@ -0,0 +1,304 @@
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <nav id="navbar">
+        <header>Documentation</header>
+        <ul>
+          <li>
+              <a class="nav-link" href="#Fruits">Fruits</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#Banana">Banana</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#Apple">Apple</a>
+          </li>
+          <li>
+              <a class="nav-link" href="#Mango">Mango</a>
+          </li>
+          <li>
+              <a class="nav-link" href="#Watermelon">Watermelon</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#Oranges">Oranges</a>
+          </li>
+          <li>
+              <a class="nav-link" href="#Grapes">Grapes</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#Apricot">Apricot</a>
+          </li>
+          <li>
+              <a class="nav-link" href="#Strawberry">Strawberry</a>
+          </li>
+          <li>
+              <a class="nav-link" href="#Pineapple">Pineapple</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#Lemon">Lemon</a>
+          </li>
+          <li>
+              <a class="nav-link" href="#Cherry">Cherry</a></li>
+          <li>
+            <a class="nav-link" href="#Kiwi">Kiwi</a>
+          </li>
+          <li><a class="nav-link" href="#Copy">Copy</a></li>
+        </ul>
+      </nav>
+      <main id="main-doc">
+        <section class="main-section" id="Fruits">
+          <header>Fruits</header>
+          <article>
+            <p>
+              Fruits are a rich source of vitamins, minerals, fibres and proteins. 
+              We should eat fresh fruits to keep ourselves healthy.
+              But we should not eat cut fruits at the market as they are very unhygienic and can make us ill.
+            </p>
+  
+            <p>
+              We should wash fruits thoroughly before eating them. While some fruits are seasonal, others are available throughout the year. 
+              Some are very expensive while the others are reasonable.
+            </p>
+            <ul>
+              <li>
+                  Mangoes and cherries are my favourite fruits. I relish them the most. I like mangoes in all forms—juice, jam and pickle. 
+                  Alphonso, Dussheri, Langda and Safeda are only some of the dozens of varieties of mangoes found in India.
+              </li>
+              <li>
+                  Eating fruits brings many benefits to the body because of the necessary elements and nutritional value, it is rich in vitamins, antioxidants, and a lot of minerals.
+              </li>
+            </ul>
+          </article>
+        </section>
```

Comment:
Use "flexbox or grid" for this section. This will have a professional impact.

Reply:
> Use "flexbox or grid" for this section. This will have a professional impact.

- [ ] Let us always try to suggest changes politely, your comment is more like an order. "I think it would be better to add.." would have been a better approach. Also, it is always important to add the WHY, this is for the student to know the reason you are requesting changes.

Reply:
- [Optional] Kindly consider the changes from the previous review  :wink: